flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/utils/AggSqlFunction.scala

```diff
@@ -37,24 +36,26 @@ import org.apache.flink.table.functions.utils.UserDefinedFunctionUtils._
   * @param name function name (used by SQL parser)
   * @param aggregateFunction aggregate function to be called
   * @param returnType the type information of returned value
+  * @param accType the type information of the accumulator
   * @param typeFactory type factory for converting Flink's between Calcite's types
   */
 class AggSqlFunction(
     name: String,
     aggregateFunction: AggregateFunction[_, _],
-    returnType: TypeInformation[_],
+    val returnType: TypeInformation[_],
+    val accType: TypeInformation[_],
     typeFactory: FlinkTypeFactory,
     requiresOver: Boolean)
-  extends SqlUserDefinedAggFunction(
+  extends SqlAggFunction(
```

Comment:
Sorry, I didn't notice the copied `SqlUserDefinedAggFunction` from Calcite. I did these change because the constructor of Calcite `SqlUserDefinedAggFunction` doesn't match the calling with additional `requiresOver`.